src/puppetlabs/puppetdb/http/aggregate_event_counts.clj

```diff
@@ -6,14 +6,15 @@
             [puppetlabs.puppetdb.query-eng :refer [produce-streaming-body]]
             [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params]]
             [clojure.tools.logging :as log]
-            [net.cgrand.moustache :refer [app]]))
+            [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.config :as conf]))
 
 (defn routes
-  [version]
+  [version config]
   (app
     [""]
     {:get (fn [{:keys [params globals]}]
-            (when (= "puppetdb" (:product-name globals))
+            (when (conf/foss? config)
```

Comment:
This seems like the simplest way to go about this. We have the static config and a set of functions that can access the pieces we care about. I also thought about an atom that we bash on startup, then this becomes (conf/foss?) and we don't have to thread it through. That might be where we want to go (or something more fancy) but it seemed like this was a better place to start, until we move more things out of shared-globals (i.e. MQ stuff, catalog-hash-debug-dir etc)